sms_statistics_service/src/cache.py

```diff
@@ -51,10 +51,10 @@ def get_active_projects(self, firestore_client):
                 json.dump([ap.to_dict() for ap in active_projects], f)
             return active_projects
 
-    def get_rapid_pro_token_for_project(self, project_name, google_cloud_credentials_file_path, rapid_pro_token_url):
-        cache_file_path = f"{self.cache_dir_path}/rapid_pro_tokens/{project_name}.txt"
+    def get_token_for_project(self, project_name, token_type, google_cloud_credentials_file_path, token_url):
+        cache_file_path = f"{self.cache_dir_path}/tokens/{project_name}/{token_type}.txt"
         try:
-            log.info(f"Attempting to read the Rapid Pro token for project '{project_name}' from the cache "
+            log.info(f"Attempting to read the {token_type} token for project '{project_name}' from the cache "
                      f"at '{cache_file_path}'")
             with open(cache_file_path) as f:
                 token = f.read()
```

Comment:
Update the log info to display the type of token

Reply:
Line 61

Reply:
Good catch, thanks